lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "version": "0.0.0",
```

Comment:
should be synchronized with latest package version, so "0.0.1" seems more suitable currently